### ISSUE TITLE
Validation loss increases after the first epoch

### ISSUE THREAD

**Issue by PVSemk** [wtomin/Multitask-Emotion-Recognition-with-Incomplete-Labels#12]
Hello!

I'm trying to implement my own pipeline to train resnet 50 for VA on Aff-wild2 only. I follow your idea of bucketization of continious values into a fixed number of classes and applying both classification CE and regression CCC losses and experimented with different weights for each. 

I tried initialization from both fer+ and imagenet

However, I've faced an issue that my validation loss increases after first epoch (along with decreasing in ccc metric). However, I can successfully overfit my model on a single video from the training subset. 

Didn't you face the same or similar problem?
